packages/app/src/components/seo-meta.js

```diff
@@ -0,0 +1,35 @@
+import Head from 'next/head'
+
+import { META } from '@/constants'
+
+export const SeoMeta = ({
+  description = META.description,
+  image = META.image,
+  logo = META.logo,
+  title = META.title,
+  twitterCardType = 'summary',
+  url = META.url
+}) => (
+  <Head>
```

Comment:
Can you please double-check

https://github.com/microlinkhq/www/blob/master/src/components/Head.js

just I want to don't miss potentially good SEO tags